driver/ixsystems/freenasapi.py

```diff
@@ -110,15 +109,6 @@ def set_api_version(self, api_version):
     def set_transport_type(self, transport_type):
         self._protocol = transport_type
 
-    def set_style(self, style):
-        """Set the authorization style for communicating with the server.
-
-           Supports basic_auth for now.
-        """
-        if style.lower() not in (FreeNASServer.STYLE_LOGIN_PASSWORD):
-            raise ValueError('Unsupported authentication style')
-        self._auth_style = style.lower()
-
     def get_url(self):
```

Comment:
style and STYLE_LOGIN_PASSWORD is used no where and not accessible by previous version by users, hence removed.